modules/gapi/perf/common/gapi_imgproc_perf_tests_inl.hpp

```diff
@@ -1072,5 +1072,91 @@ PERF_TEST_P_(RGB2YUV422PerfTest, TestPerformance)
 
 //------------------------------------------------------------------------------
 
+// TODO: move to the separate file modules/gapi/perf/common/gapi_video_perf_tests_inl.hpp
+PERF_TEST_P_(OptFlowLKPerfTest, TestPerformance)
+{
+    int maxLevel = 5, flags = 0, format = 1;
+    double minEigThreshold = 1e-4;
+
+    Point2fVector inPts, outPtsOCV, outPtsGAPI;
+    UcharVector   outStatusOCV, outStatusGAPI;
+    FloatVector   outErrOCV, outErrGAPI;
+
+    auto outTupleOCV  = std::make_tuple(std::ref(outPtsOCV), std::ref(outStatusOCV),
+                                        std::ref(outErrOCV));
+    auto outTupleGAPI = std::make_tuple(std::ref(outPtsGAPI), std::ref(outStatusGAPI),
+                                        std::ref(outErrGAPI));
+
+    cv::GComputation c = runOCVnGAPIOptFlowLK(*this, { get<1>(GetParam()), format,
+                                                       get<2>(GetParam()), get<3>(GetParam()),
+                                                       get<4>(GetParam()), maxLevel,
+                                                       get<5>(GetParam()), flags, minEigThreshold,
+                                                       get<6>(GetParam()) },
```

Comment:
I recommend to use `tie()` instead. It is more type-safe than this.

Reply:
I can't agree: if we use tie() here, we will have to predefine all the variables, but actually we don't really need them

Reply:
Explained offline

Reply:
OK, the use of get<> is removed